Modules/Relationships/src/Relationships.ConsumerApi/Controllers/RelationshipsController.cs

```diff
@@ -87,9 +86,6 @@ public async Task<IActionResult> CreateRelationship(CreateRelationshipCommand re
     [ProducesError(StatusCodes.Status404NotFound)]
     public async Task<IActionResult> AcceptRelationship([FromRoute] string id, [FromBody] AcceptRelationshipRequest request, CancellationToken cancellationToken)
     {
-        var peerOfActiveIdentityInRelationshipResponse = await _mediator.Send(new GetPeerOfActiveIdentityInRelationshipQuery { Id = id }, cancellationToken);
-        await EnsurePeerIsNotToBeDeleted(peerOfActiveIdentityInRelationshipResponse.IdentityAddress, cancellationToken);
-
         var response = await _mediator.Send(new AcceptRelationshipCommand { RelationshipId = id, CreationResponseContent = request.CreationResponseContent }, cancellationToken);
```

Comment:
Like I said in another comment, this doesn't really make sense to me from the design point of view. Just wanted to let @tnotheis know.

Reply:
Waiting on Timo's answer.